korangar/src/main.rs

```diff
@@ -307,9 +314,23 @@ fn main() {
             .map(|_| picker_renderer.create_render_target())
             .collect::<Vec<<PickerRenderer as Renderer>::Target>>();
 
-        let mut shadow_targets = (0..max_frame_count)
-            .map(|_| shadow_renderer.create_render_target(shadow_detail.get().into_resolution()))
-            .collect::<Vec<<ShadowRenderer as Renderer>::Target>>();
+        let mut directional_shadow_targets = (0..max_frame_count)
+            .map(|_| directional_shadow_renderer.create_render_target(shadow_detail.get().into_resolution()))
+            .collect::<Vec<<DirectionalShadowRenderer as Renderer>::Target>>();
+
+        let mut point_shadow_targets = (0..max_frame_count)
+            .map(|_| {
+                (0..NUMBER_OF_POINT_LIGHTS_WITH_SHADOWS)
+                    // TODO: Replace with shadow detail
```

Comment:
Do you still want to do that in this PR?

Reply:
I actually didn't mean to but now that you called me out, yes :D